public/js/thebuggenie.js

```diff
@@ -55,6 +55,10 @@ define(['thebuggenie/tools', 'thebuggenie/tbg', 'domReady', 'jquery', 'mention']
                     TBG.Main.Profile.clearPopupsAndButtons();
                     e.stopPropagation();
                 });
+                jQuery("img[data-src]:not([data-src-processed])").each(function(){
+                    var $img = jQuery(this);
+                    $img.attr('src', $img.data('src')).data('src-processed', true);
+                });
```

Comment:
When images are loaded via ajax, then the images come as is in src. Asynchronous image loading didn't make much sense to us here.